### ISSUE TITLE
DOC605 error is not specific enough, making its fixation a stuggle

### ISSUE THREAD

**Issue by ssbarnea** [jsh9/pydoclint#211]
I do have a special care with a Class with more than 100 attributes and finding which attributes are not in order seems like finding the needle in the haystack. The error should at least give the first culprit in order to make it actionable.

```
DOC604: Class `ClassName`: Attributes are the same in docstring and class def, but are in a different order.  (Please read https://jsh9.github.io/pydoclint/checking_class_attributes.html on how to correctly document class attributes.)
```

**Comment by jsh9** [comment]
Hi @ssbarnea ,

Is this the instructions you are looking for?  (ChatGPT helped me write this script; I haven't thoroughly tested it.)

```python
def generate_reordering_steps(a: list[str], b: list[str]) -> list[str]:
    if a == b:
        return ["b is already in the correct order."]
    
    index_map = {value: i for i, value in enumerate(b)}
    b_sorted = b[:]
    
    steps = []
    
    for i, expected in enumerate(a):
        current_index = index_map[expected]
        if current_index != i:
            steps.append(
                f"Swap '{b_sorted[i]}' at index {i} with '{b_sorted[current_index]}' at index {current_index}."
            )
            # Perform swap in the copy
            b_sorted[i], b_sorted[current_index] = b_sorted[current_index], b_sorted[i]
            # Update index map
            index_map[b_sorted[current_index]] = current_index
            index_map[b_sorted[i]] = i
    
    return steps

# Example usage:
a = list(range(10))
b = [8, 7, 2, 6, 4, 5, 3, 1, 0, 9]


steps = generate_reordering_steps(a, b)
for step in steps:
    print(step)
```

The output is:
```
Swap '8' at index 0 with '0' at index 8.
Swap '7' at index 1 with '1' at index 7.
Swap '6' at index 3 with '3' at index 6.
```

And both time and space complexity is O(n) (again, according to ChatGPT, and I haven't thoroughly tested it).
